portafly/src/pages/Accounts.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useDocumentTitle, Loading, AccountsTable } from 'components'
+import {
+  Alert,
+  PageSection,
+  PageSectionVariants,
+  TextContent,
+  Text
+} from '@patternfly/react-core'
+import { useGetDeveloperAccounts } from 'dal/accounts'
+import { useTranslation } from 'i18n/useTranslation'
+
+const Accounts: React.FunctionComponent = () => {
+  const { t } = useTranslation('accounts')
+  useDocumentTitle(t('page_title'))
+
+  const { accounts, error, isPending } = useGetDeveloperAccounts()
+
+  return (
+    <>
+      <PageSection variant={PageSectionVariants.light}>
+        <TextContent>
+          <Text component="h1">{t('body_title')}</Text>
+        </TextContent>
+      </PageSection>
+
+      <PageSection>
+        {isPending && <Loading />}
+        {error && <Alert variant="danger" title={error.message} />}
```

Comment:
Should "danger" be included in the strings file , maybe in shared?

Reply:
It is a fixed input controlled by `Alert`:
```jsx
variant?: 'success' | 'danger' | 'warning' | 'info' | 'default';
```
Not really a string susceptible to the locale, so IMO it's supposed to be "hardcoded".